tests/cli/test_cli.py

```diff
@@ -743,12 +745,11 @@ def test_upload1(ctx):
     )  # Directory listing with 2 headers lines and one file
 
     # Upload a file that exceeds the disk quota
-    username = _run_command([cl, 'uinfo', '-f', 'user_name'])
-    prev_dq = _run_command([cl, 'uinfo', username, '-f', 'disk']).split(' ')[2]
-    _run_command([cl, 'uedit', username, '--disk-quota', '2'])
+    prev_dq = _run_command([cl, 'uinfo', 'codalab', '-f', 'disk']).split(' ')[2]
+    _run_command([cl, 'uedit', 'codalab', '--disk-quota', '2'])
     # expect to fail when we upload something more than 2 bytes
     _run_command([cl, 'upload', 'codalab.png'], expected_exit_code=1)
-    _run_command([cl, 'uedit', username, '--disk-quota', prev_dq])
+    _run_command([cl, 'uedit', 'codalab', '--disk-quota', prev_dq])
```

Comment:
we don't need this line anymore

Reply:
so `__exit__` runs at the end of every module, not the entire suite of tests?

Reply:
I believe so. You can test it out by printing something there and/or following the code for test modules.

Reply:
yep it does